src/T2DMap.cpp

```diff
@@ -613,6 +613,65 @@ void T2DMap::addSymbolToPixmapCache(const QString key, const bool gridMode)
     }
 }
 
+/*
+ * Helper used to size fonts establishes the font size to use to draw the given
+ * sample text centralized inside the given boundary (or rather the boundary
+ * reduced by the margin (0-40) as a percentage). This margin is defaulted to
+ * 20%.
+ */
+bool T2DMap::sizeFontToFitTextInRect( QFont & font, const QRectF & boundaryRect, const QString & text, const quint8 percentageMargin )
+{
+    QFont _font = font;
+
+    if (percentageMargin > 80) {
+        qWarning() << "T2DMap::sizeFontToFitTextInRect(...) percentage margin" << percentageMargin << "exceeded maximum (40%) !";
```

Comment:
It's checking 80, not 40?

Reply:
Yeah that needs revision - it overrides the normal margin of 20% - or it would if I had put it in :woman_facepalming:   (10% on each side, top and bottom) margin around the text and setting it too large will make the text too small (a little blob in the middle of the area) to be useful...

Reply:
Fixed.